cursorless-talon/src/primitive_target.py

```diff
@@ -1,20 +1,22 @@
+from typing import Any
+
 from talon import Module
 
 mod = Module()
 
-BASE_TARGET = {"type": "primitive"}
+BASE_TARGET: dict[str, Any] = {"type": "primitive"}
 IMPLICIT_TARGET = {"type": "primitive", "isImplicit": True}
 
 
 modifiers = [
     "<user.cursorless_position>",  # before, end of
-    "<user.cursorless_selection_type>",  # token, line, file
     "<user.cursorless_head_tail>",  # head, tail
     "<user.cursorless_containing_scope>",  # funk, state, class
     "<user.cursorless_subtoken_scope>",  # first past second word
     "<user.cursorless_surrounding_pair>",  # matching/pair [curly, round]
     "<user.cursorless_to_raw_selection>",  # just
-    # "<user.cursorless_matching_paired_delimiter>",  # matching
+    "<user.cursorless_delimiter_inclusion>",  # inside, bounds
+    "<user.cursorless_delimiter_range>",  # leading, trailing
```

Comment:
It is a bit funny that we use "delimiter" for both of these things, but we treat them completely differently 🤔.  Might be a bit late to try to tease these apart, as I believe some of this terminology is already baked into the csv.  Prob worth a quick chat today

Reply:
We can definitely have a look at it